Kernel/FourVelocity.m

```diff
@@ -110,46 +138,53 @@
 (*Circular Case*)
 
 
-KerrGeoVelocityDarwin[a_,p_,(0|0.),x_,initPhases_,index_ ]:= Module[{ut,ur,u\[Theta],u\[Phi], MinoVelocities,ut1,ur1,u\[Theta]1,u\[Phi]1},
+KerrGeoVelocityDarwin[a_,p_,(0|0.),x_,initPhases_]:= Module[{ut,ur,u\[Theta],u\[Phi], MinoVelocities,utContra,urContra,u\[Theta]Contra,u\[Phi]Contra,
```

Comment:
As above, better not using `(0|0.)` here

Reply:
I've now used ?PossibleZeroQ for consistency with the rest of the package.